package.json

```diff
@@ -3,11 +3,11 @@
   "version": "0.0.1",
   "description": "Next.js & React app",
   "scripts": {
-    "dev": "next -p 8080",
+    "dev": "next",
```

Comment:
Can you plz specify explicitly which port are we running on?
And update `Dockerfile` and `babajka-deploy` repo accordingly.

Reply:
Fixed it.